src/api/services/timecardService.js

```diff
@@ -13,15 +13,15 @@ export const getTimecard = async (params) => {
 
 export const getTimeEntries = async (params) => {
   try {
-    return await api.get(baseUrl + 'resources/time-entry/', params);
+    return await api.get(baseUrl + 'resources/time-entries/', params);
```

Comment:
This string literal is repeated a few times. It would be better to extract it into a constant

Reply:
I agree this should be configurable value, but we will do it as separate ticket, this should be aligned with server changes.